plugins/pulp_rpm/plugins/importers/yum/depsolve.py

```diff
@@ -72,9 +72,6 @@ def __cmp__(self, other):
                       a positive value if self is greater than other.
         :rtype:       int
         """
-        if self.name != other.name:
```

Comment:
Ah, that is a good point.  I wasn't paying attention there.  I'll add that block back.